widget/ui/src/components/Toast/Toast.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import type { Type } from '../Alert/Alert.types';
+import type { Meta } from '@storybook/react';
+
+import React from 'react';
+
+import { Button } from '../Button';
+
+import { Toast, ToastProvider, useToast } from '.';
+
+export default {
+  title: 'Components/Toast',
+  component: Toast,
+  argTypes: {
+    type: {
+      name: 'type',
+      defaultValue: 'success',
+      control: {
+        type: 'select',
+        options: ['success', 'warning', 'error', 'info', 'loading'],
+      },
+    },
+    title: {
+      name: 'title',
+      defaultValue: 'Please change your wallet network to Polygon',
+      control: {
+        type: 'text',
+      },
+    },
+  },
+} as Meta<typeof Toast>;
+
+export const Main = ({
+  horizontal,
+  ...args
+}: {
+  type: Type;
+  title: string;
+  horizontal: 'left' | 'right';
+}) => {
+  return (
+    <ToastProvider
+      container={document.body}
+      anchorOrigin={{
+        horizontal,
+        vertical: 'bottom',
+      }}>
+      <ToastComponent {...args} />
+    </ToastProvider>
+  );
+};
+
+const ToastComponent = (args: { type: Type; title: string }) => {
+  const { addToast } = useToast();
+
+  return (
+    <div>
+      <Button
+        variant="contained"
+        type="success"
+        onClick={() => addToast({ ...args })}>
```

Comment:
This button is not displayed for testing

Reply:
fixed.